src/modules/form/components/DynamicField.tsx

```diff
@@ -0,0 +1,130 @@
+import { Grid, Stack, Paper, Typography } from '@mui/material';
+import React, { ReactNode } from 'react';
+
+import { FieldType, Item } from '../types';
+
+import FormSelect from './FormSelect';
+
+import DatePicker from '@/components/elements/input/DatePicker';
+import OrganizationSelect from '@/components/elements/input/OrganizationSelect';
+import ProjectSelect from '@/components/elements/input/ProjectSelect';
+import TextInput from '@/components/elements/input/TextInput';
+
+interface Props {
+  item: Item;
+  itemChanged: (uid: string, value: any) => void;
+  value: any;
+  children?: (item: Item) => ReactNode;
+}
+
+const ItemGroup = ({
+  children,
+  title,
+}: {
+  children: ReactNode;
+  title?: string;
+}) => (
+  <Paper
+    sx={{
+      ml: 1,
+      mt: 2,
+      py: 1,
+      px: 2,
+      borderLeft: 3,
+      borderLeftColor: 'primary.light',
+    }}
+  >
+    {title && <Typography sx={{ mb: 1 }}>{title}</Typography>}
+    <Stack direction='column' spacing={1}>
+      {children}
+    </Stack>
+  </Paper>
+);
+
+const DynamicField: React.FC<Props> = ({
```

Comment:
`DynamicField` is where we map a given Item to a Component, based on it's type ("string", "choice", etc)